demo/predict-taxi-trip-duration-nb/Dockerfile

```diff
@@ -1,10 +1,137 @@
-FROM centos:8
+# https://github.com/docker-library/openjdk/blob/61bfbb30ae5a0395f08d41107b86bcce90d70ba6/11/jre/slim-bullseye/Dockerfile
+FROM openjdk:11.0.13-jre-slim-bullseye
 
-LABEL org.opencontainers.image.source https://github.com/4paradigm/OpenMLDB
+# install python 3.8
+# from https://github.com/docker-library/python/blob/9242c448c7e50d5671e53a393fc2c464683f35dd/3.8/bullseye/slim/Dockerfile
+ENV GPG_KEY E3FF2839C048B25C084DEBE9B26995E310250568
+ENV PYTHON_VERSION 3.8.12
 
-RUN yum install -y python38 python38-pip python38-numpy libgomp java-1.8.0-openjdk-headless && \
-    yum clean all && \
-    pip3 install lightgbm tornado requests pandas openmldb
+RUN set -ex \
+	\
+	&& savedAptMark="$(apt-mark showmanual)" \
+	&& apt-get update && apt-get install -y --no-install-recommends \
+		dpkg-dev \
+		gcc \
```

Comment:
python compile dependency may not included in final image
can we use multi-stage build ?

Reply:
The python compile dependency deps will be auto purged by apt-mark.